CHANGELOG.md

```diff
@@ -82,6 +82,8 @@
   accepting negative values.
 - Fixed #1754 - net: traffic blocks when running ingress UDP performance tests
   with very large buffers.
+- Fixed `devtool build` to build jailer binary for `musl` only targets. Building 
```

Comment:
This should be in the UNRELEASED section of the changelog

Reply:
Wondering if this will be moved from "UNRELEASED" to "Fixed" section before release? Anyways I have made the change. Working on the test now.

Reply:
It will be moved from Unreleased in the future. The v0.22.0 release already exists so these new changes will be included in a future release.

Reply:
Are we actually *fixing* something through the PR? Or should this line be expressed as *updated* or something else? 

What do you think?

Reply:
Hi @gc-plp and @alindima , modified this line to "changed" instead of "fixed" and moved it to Changed sub-section of UnReleased.